crowbar_framework/app/models/batch/export.rb

```diff
@@ -17,6 +17,8 @@
 
 module Batch
   class Export < Base
+    attr_accessor :path
```

Comment:
otherwise I wouldn't be able to set `path` which means I cannot set a custom path for the yml output (the backup path in our case)